operator/src/main/resources/scripts/startServer.sh

```diff
@@ -110,7 +110,12 @@ function waitForShutdownMarker() {
   # Wait forever.   Kubernetes will monitor this pod via liveness and readyness probes.
   #
   trace "Wait indefinitely so that the Kubernetes pod does not exit and try to restart"
+  logged=false
   while true; do
+    if [ "$logged" != 'true' ] && grep -q "BEA-141335" ${SERVER_OUT_FILE} ; then
+      trace "WebLogic server failed to start due to missing or invalid situational configuration files. Please check ${SERVER_OUT_FILE} for details"
+      logged=true
+    fi
```

Comment:
trace "WebLogic server failed to start due to missing or invalid situational configuration files. Please check ${SERVER_OUT_FILE} for details"
	     
-->

trace "WebLogic server failed to start due to missing or invalid situational configuration files, which may be due to invalid configOverride templates (these are specified via the configOverride attribute in the Domain custom resource).  For details, please search your pod log or ${SERVER_OUT_FILE} for the keyword 'situational'."

Reply:
Fixed as suggested. Thanks.

Reply:
Text LGTM.